build.gradle

```diff
@@ -213,7 +213,7 @@ task contract(type: Test) {
 }
 
 task pullTestAsset(type: Exec, description: 'Installs Yarn dependencies.') {
-  commandLine './bin/pull-latest-ccd-files.sh', 'master'
+  commandLine './bin/pull-latest-ccd-files.sh', 'CIV-5708-second-PR'
```

Comment:
This is temporary for the purpose of testing this PR, I assume.

Don't forget to remove this change once you're finished.

Reply:
Thanks! Now removed and reverted to master